.github/workflows/codeql.yml

```diff
@@ -24,14 +24,14 @@ jobs:
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
+      uses: github/codeql-action/init@v2
       with:
          languages: javascript
         
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
+      uses: github/codeql-action/autobuild@v2
```

Comment:
Since this project is TypeScript, there's no need for this step. Harmless to keep it in, but it can be removed since it's a noop.

Reply:
Thanks for chiming in, that's good to know! 

I'm going to leave it for now since I want to isolate this one change, but we'll circle back to that in a subsequent PR. May as well delete it if it's really just a noop. 👍